front/temporal/upsert_queue/activities.ts

```diff
@@ -146,3 +150,129 @@ export async function upsertDocumentActivity(
     upsertContext: upsertQueueItem.upsertContext || undefined,
   });
 }
+
+export async function upsertTableActivity(
```

Comment:
Pretty sure we could have factorized a bit this code but given that we will very likely move it to a dedicated queue, I guess it's okay.